Horvat Diana Adriana/lab 5 pb 3,10/main.c

```diff
@@ -0,0 +1,75 @@
+#include <stdio.h>
+#include <stdlib.h>
+int n;
+int *nrbanc;
+void numarMin_bancnote (int *nrbanc, int n)
+{
+    *nrbanc=0;
+    if(n>=500)
```

Comment:
If you would have an array in which you'd put all these static values (`500, 200 etc.`) you could iterate through that array and then apply this while loop for each of the values in that array => you won't have so much duplicate code (so many while loops...)